textworld/agents/walkthrough.py

```diff
@@ -26,7 +26,7 @@ def reset(self, env):
             raise NameError(msg)
 
         # Load command from the generated game.
-        self._commands = iter(env.game.quests[0].commands)
+        self._commands = iter(env.game.quests[-1].commands)
```

Comment:
I need to rethink this. It is not guaranteed the last quest will have all the commands needed to solve the whole game. It should be the case for games generated via the nonlinear chaining function but not for handcrafted games.